lib/files.go

```diff
@@ -125,6 +132,7 @@ func WriteLines(lines []string, path string) (err error) {
 }
 
 // ReadLines : Read a whole file into the memory and store it as array of lines
+// @TODO This isn't used - should we keep it?
```

Comment:
From https://github.com/warrensbox/terraform-switcher/pull/436#discussion_r1615333045

Must be considered in review - either remove the TODO or remove the method - same with WriteLines

Reply:
From @yermulnik 

As we've learned there are loads (or at least some 🤪) of the projects that use `tfswitch` as a lib or re-use its code as-is, we may probably need to announce removal of functions/methods/whatever in advance and give consumers time to at least next release to migrate/replace.
So what I'd suggest is to leave this as is and get this removed any time later after any major release and in case any of use stumble upon it by an accident or on purpose (and hence maybe rephrase it like `... - should remove anytime after 2.0.0`). IMHO.

Reply:
Yeah maybe a depreciation note. 
Perhaps Release 2.0

Reply:
Done :)